### DISCUSSION TITLE
How do I see the toots people made about trending stories?

### DISCUSSION THREAD

**Discussion by ghost** [mastodon/mastodon#17300] [Q&A] [answered]
All I see is that "23 people are talking about it" but I can't see their toots. Is this a missing feature or am I just blind?

![afbeelding](https://user-images.githubusercontent.com/2829538/149658725-5eae6eeb-8427-4282-a873-98ec62b66ee3.png)

**Comment by ClearlyClaire** [comment, marked as answer]
It's not possible.

**Reply by Gargron** [reply]
It’s a feature in development. It’s currently not possible but might be added if there’s demand

**Reply by ghost** [reply]
Where can my interest be recorded? I can't find an issue to "thumbs up" or "+1", only a PR https://github.com/mastodon/mastodon/pull/16917
